### PR TITLE
feat: add AppHeader and AppMenu

### PR DESCRIPTION
**Pull request by gularsson**

### CODE REVIEW

File: src/components/ThemeSwitcher/index.tsx
Comment (dannelundqvist) [inline src/components/ThemeSwitcher/index.tsx:1]: Would be nice with empty line between const declaration and return statement for readability.

File: src/components/ViewHeader/index.tsx
Comment (dannelundqvist) [inline src/components/ViewHeader/index.tsx:1]: Would be nice with empty line between interface and function for readability.

File: src/components/index.tsx
Comment (dannelundqvist) [inline src/components/index.tsx:1]: I think it would be nice to group some of these into one folder. I.e AppMenu, AppHeader, ThemeSwitcher

**Review by dannelundqvist — CHANGES_REQUESTED** [review]

File: src/components/index.tsx
Comment (gularsson) [reply src/components/index.tsx:1]: `AppHeader` is a composite view which uses, `AppMenu` and `ThemeSwitcher` are never (not now) used outside of it.

**Review by dannelundqvist — APPROVED** [review]
Super!

**Pull request merged**